src/notation/view/notationviewinputcontroller.cpp

```diff
@@ -29,13 +29,14 @@
 
 #include "log.h"
 #include "commonscene/commonscenetypes.h"
-#include "internal/abstractelementpopupmodel.h"
+#include "inspector/iinspectorcontroller.h"
 
 using namespace mu;
 using namespace mu::notation;
 using namespace mu::engraving;
 using namespace mu::actions;
 using namespace mu::commonscene;
+using namespace mu::inspector;
```

Comment:
I think this is not necessary; in this file you're only using `inspectorController()`, which is a method of NotationViewInputController (generated by INJECT), and thus does not need the inspector namespace.